.eslintrc.cjs

```diff
@@ -12,21 +12,26 @@ module.exports = {
   plugins: ['react'],
   rules: {
     'spaced-comment': 'off',
+    'react/prop-types': 'off',
     'react/button-has-type': 'off',
-    "no-use-before-define": "off",
+    'no-use-before-define': 'off',
```

Comment:
I don't agree with you in removing the following rules :

- '**no-use-before-define': 'off'**
- **'import/no-named-as-default-member': 'off'**
- **'import/no-named-as-default': 'off'**
disabling them may lead to avoidable errors in our codes

Reply:
Great insight @jcharles921